apps/metadata-editor-e2e/src/e2e/my-org.cy.ts

```diff
@@ -1,9 +1,14 @@
 describe('my-org', () => {
   beforeEach(() => {
     cy.loginGN('barbie', 'p4ssworD_', false)
+    cy.intercept({
+      method: 'GET',
+      url: 'http://localhost:4200/geonetwork/srv/api/userselections/0/101',
+    }).as('dataGetFirst')
     cy.visit(`/records/my-org`)
     cy.get('md-editor-dashboard-menu').find('a').first().click()
-    cy.get('main').children('div').first().children('div').eq(1).as('linkGroup')
+    cy.wait('@dataGetFirst').its('response.statusCode').should('equal', 200)
```

Comment:
I'm not sure I understand this assertion; first, it's counter-intuitive to have one in a `beforeEach` block, and then it looks like it's checking that favorites are correctly queried for the current user: why is it done, and why here?

Reply:
The beforeEach sets a block as linkGroup, but sometimes, the xhr response hasn't finished and the test was failing on this line `cy.get('main').children('div').first().children('div').eq(1).as('linkGroup')`

Before XHR : 
![Screenshot from 2023-10-23 11-33-08](https://github.com/geonetwork/geonetwork-ui/assets/39771412/bacaae9e-abe8-4e0c-b0c3-7a3677b6131f)

After XHR : 
![image](https://github.com/geonetwork/geonetwork-ui/assets/39771412/f35a523e-2157-411a-a273-2301051375a0)